pages/workflow/PKPWorkflowHandler.inc.php

```diff
@@ -314,6 +314,12 @@ function setupTemplate($request) {
 		$templateMgr->assign('stageId', $stageId);
 		$templateMgr->assign('submissionStageId', $submission->getStageId());
 		$templateMgr->assign('workflowStages', $workflowStages);
+		
+		$submissionDao = Application::getSubmissionDAO();
+		$router = $request->getRouter();
+		$contextId = $router->getContext()->getId();
+		$newestVersion = $submissionDao->getLatestRevisionId($submission->getId(), $contextId);
+		$templateMgr->assign('newestVersion', $newestVersion);
```

Comment:
Hmm.. If that default version = 1 in get Authors would get the latest submission version this would eventually not be necessary?